shared/js/initApp.js

```diff
@@ -14,7 +14,7 @@ window.apps = {
   // Legacy Blockly initialization that was moved here from _blockly.html.haml.
   // Modifies `appOptions` with some default values in `baseOptions`.
   // TODO(dave): Move blockly-specific setup function out of shared and back into dashboard.
-  setupBlockly: function () {
```

Comment:
Re: this file, know it wasn't changed in this PR, jumping in to this file, I'm not immediately sure whether it's specific to all Droplet/Blockly levels, all `apps/` levels, or any level. A file-level comment could be helpful up top just to clarify